doc/options.rst

```diff
@@ -0,0 +1,115 @@
+        * *abort_on_first_run_crash* in [true, false]. If true, *SMAC* will abort if the first run of the target algorithm crashes. Default: true.
+        * *execdir* specifies the path to the execution-directory. Default: ".".
+        * *deterministic* in [true, false]. If true, the optimization process will be repeatable. Default: false 
+        * *overall_obj* is PARX, where X is an integer defining the penalty imposed on timeouts (i.e. runtimes that exceed the *cutoff-time*). Default: PAR10.
+        * *memory_limit* is the maximum available memory the target-algorithm can occupy before being cancelled.
+        * *tuner-timeout* is the maximum amount of CPU-time used for optimization. Default: inf.
+        * *wallclock_limit* is the maximum amount of wallclock-time used for optimization. Default: inf.
+        * *runcount_limit* is the maximum number of algorithm-calls during optimization. Default: inf.
+        * *minR* is the minimum number of calls per configuration. Default: 1
+        * *maxR* is the maximum number of calls per configuration. Default: 2000
+        * *always_race_default* indicates that new incumbents are always raced against default configuration. Default: false
+        * *instance_file* specifies the file with the training-instances.
+        * *test_instance_file* specifies the file with the test-instances.
+        * *feature_file* specifies the file with the instance-features
+        * *output_dir* specifies the output-directory for all emerging files, such as logging and results. Default: "smac3-output_YEAR-MONTH-DAY_HOUR:MINUTE:SECOND"
+        * *shared_model*:  Default: false
+        * *initial_incumbent*: in [DEFAULT, RANDOM]. DEFAULT is the default from the PCS. Default: DEFAULT.
+
+.. _paramcs:
+
+Parameter Configuration Space (PCS)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Parameter Configuration Space (PCS) defines the legal ranges of the
+parameters to be optimized and their default values. In the examples-folder you
+can find several examples for PCS-files. Generally, the format is:
+
+.. code-block:: bash
+
+        parameter_name categorical {value_1, ..., value_N} [default value]
+        parameter_name ordinal {value_1, ..., value_N} [default value]
+        parameter_name integer [min_value, max_value] [default value]
+        parameter_name integer [min_value, max_value] [default value] log
+        parameter_name real [min_value, max_value] [default value]
+        parameter_name real [min_value, max_value] [default value] log
+
+        # Conditionals:
+        child_name | condition [&&,||] condition ...
+
+        # Condition Operators: 
+        # parent_x [<, >] parent_x_value (if parameter type is ordinal, integer or real)
+        # parent_x [==,!=] parent_x_value (if parameter type is categorical, ordinal or integer)
+        # parent_x in {parent_x_value1, parent_x_value2,...}
+
+        # Forbiddens:
+        {parameter_name_1=value_1, ..., parameter_name_N=value_N}
+
+.. note::
```

Comment:
Sure? Can't it read both the old and the new format? Or do you mean that it is more restrictive than the implementation of the *new* PCS format in SMAC2?